virtualization/Docker/Dockerfile.dev

```diff
@@ -25,7 +25,7 @@ RUN virtualization/Docker/setup_docker_prereqs
 # Install hass component dependencies
 COPY requirements_all.txt requirements_all.txt
 RUN pip3 install --no-cache-dir -r requirements_all.txt && \
-    pip3 install --no-cache-dir mysqlclient psycopg2 uvloop cchardet
+    pip3 install --no-cache-dir mysqlclient psycopg2 cchardet
```

Comment:
I don't like it but I'm down to remove `uvloop` from the Docker container. Make sure we raise an issue with aiocoap and/or uvloop. Not sure where the problem is.